pkg/cdn/metadata/mongo/mongo.go

```diff
@@ -0,0 +1,40 @@
+package mongo
+
+import "github.com/globalsign/mgo"
+
+// MetadataStore represents a Mongo backed metadata store.
+type MetadataStore struct {
+	session *mgo.Session
+	db      string
+	col     string
+	url     string
+}
+
+// NewMongoStorage returns an unconnected Mongo backed storage
+// that satisfies the Storage interface.  You must call
+// Connect() on the returned store before using it.
+func NewMongoStorage(url, dbName string) *MetadataStore {
```

Comment:
nit: this name "stutters" because it will be called with `mongo.NewMongoStorage`. can you call it `NewStorage` instead?